dev/Deployment/Deployment.idl

```diff
@@ -1,46 +1,40 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-namespace Microsoft.Windows.ApplicationModel.WindowsAppSDK
+namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
 {
     /// Represents the current Deployment status of the WindowsAppRuntime
+    [experimental][flags]
```

Comment:
Why is this 'flags'? If it's flags, the values need to be single-bit values - 0x1, 0x2, 0x4, etc. I think instead this is just a linear list, and the 'flags' is not required.

Reply:
I was considering that there could be concurrent states, such as PackageInstallRequired and PackageInstallFailed or PackageInstallPending and have more specific states indicated later on similar to PackageStatus. But we can also infer that - if the status is PackageInstallFailed then it should be obvious that a package install is required, and same with Pending or other states. So I removed flags and kept as is.